README.md

```diff
@@ -15,12 +15,13 @@
   * [Adding or Removing Windows Features](#adding-or-removing-windows-features)
   * [Website Installation Example](#website-installation-example)
   * [Optionally Configure the DSC LCM RefreshMode](#optionally-configure-the-dsc-lcm-refreshmode)
-5. [Limitations](#limitations)
+5. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
```

Comment:
Copied this to aline with other module documentation, but I don't know that it's valid given this section does not explain 'what the module is doing and how'